tools/utility/dockerise/platform.py

```diff
@@ -74,33 +77,31 @@ def build(repository, platform):
             current = stat.S_IMODE(os.lstat(p).st_mode)
             os.chmod(p, current & ~(stat.S_IWGRP | stat.S_IWOTH))
 
-    # Pull the latest version
-    err = pty.spawn(["docker", "pull", remote_tag])
-    if err != 0:
-        cprint("Docker pull returned exit code {}".format(err), "red", attrs=["bold"])
-        exit(err)
+    # Create a buildx instance for building this image
+    builder_name = "{}_{}".format(defaults.image, platform)
+    if subprocess.call(["docker", "buildx", "inspect", builder_name], stderr=DEVNULL, stdout=DEVNULL) != 0:
+        subprocess.call(["docker", "buildx", "create", "--name", builder_name], stderr=DEVNULL, stdout=DEVNULL)
```

Comment:
You should probably replace these `.call()`s with `.run().returncode`s, per https://github.com/NUbots/NUbots/pull/374#issuecomment-783741219

Reply:
I should probably should have picked that up in the other PR, but oh well

Reply:
You're making the call more complicated for no gain. It's not like `call` is deprecated